src/markdown-pages/landing-es.md

```diff
@@ -13,19 +13,19 @@ tagline:
   heading: es A tagline highlights your approach
   text: es The tagline should inspire confidence and interest, focusing on the value that your overall approach offers to your audience. Use a heading typeface and keep your tagline to just a few words, and don’t confuse or mystify. Use the right side of the grid to explain the tagline a bit more. What are your goals? How do you do your work? Write in the present tense, and stay brief here. People who are interested can find details on internal pages.
 graphics:
-  - image: https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/dist/img/circle-124.png
+  - image: http://placehold.it/150g
```

Comment:
Used gray placeholder images so the previous blue (which was uswds primary) did not cause any confusion when looking at the design